corehq/apps/es/es_query.py

```diff
@@ -160,12 +159,14 @@ def __init__(self, index=None, debug_host=None, es_instance_alias=ES_DEFAULT_INS
         self._default_filters = deepcopy(self.default_filters)
         self._aggregations = []
         self.es_instance_alias = es_instance_alias
-        self.es_query = {"query": {
-            "filtered": {
-                "filter": {"and": []},
-                "query": queries.match_all()
+        self.es_query = {
+            "query": {
+                "bool": {
+                    "filter": [],
```

Comment:
As a sanity check, have you confirmed that the `bool` filter can handle an empty list or a list with a single element?

Reply:
You mean whether this default query with no elements in under `query.bool.filter` matches all docs? Yes, it does.

Reply:
perfect, thanks